src/components/Contacts/ContactPartnershipStatus/ContactLateStatusLabel/getDonationLateStatus.test.ts

```diff
@@ -25,28 +25,54 @@ describe('getDonationLateStatus', () => {
   });
 
   it('uses the later date when both are provided', () => {
-    const lateAt = '2019-12-22';
-    const pledgeStartDate = '2019-12-01';
-    expect(getDonationLateStatus(lateAt, pledgeStartDate)).toEqual(
-      ContactLateStatusEnum.LateMoreThirty,
+    expect(getDonationLateStatus('2019-12-22', '2019-09-01')).toEqual(
+      ContactLateStatusEnum.LateLessThirty,
```

Comment:
I liked it when you had variables, as it become easier to understand which is the lateAt date.